taskcluster/firefox_tv_taskgraph/target_tasks.py

```diff
@@ -7,11 +7,27 @@
 from taskgraph.target_tasks import _target_task, standard_filter
 
 
+def tag_filter(target_release_type, task, params):
+    release_type = task.attributes.get("release-type")
+    return standard_filter(task, params) and (release_type is None or release_type == target_release_type)
+
+
 @_target_task('default')
 def target_tasks_default(full_task_graph, parameters, graph_config):
     """Target the tasks which have indicated they should be run on this project
     via the `run_on_projects` attributes."""
-    def filter(task, params):
-        return standard_filter(task, params)
+
+    if parameters["tasks_for"] == "github-release":
```

Comment:
Like Tom said, it's better to move this `if` statement at the beginning of the pipeline, in https://github.com/mozilla-mobile/fenix/pull/5564/files#diff-53e691d50a0b26d43d925c2150567e76R52. 

Can be done in a followup.